Makefile

```diff
@@ -6,7 +6,27 @@ default init-env init create-user up down stop migrate:
 migrate-%:
 	make -C pipeline migrate-$*
 
-# docker-build-% docker-status
+docker-build:
+	rm -rf src/back/auth/build_release
```

Comment:
А для чего очищать build_release ? Чтобы в докер не попало? А запись в .dockerignore от этого не помогает? Может добавить туда ещё что-то такое  build_release/

Reply:
Да, в докер если попадает папка build_release, то ошибка с кешем cmake:
```
0.426 cmake -B build_release -DCMAKE_INSTALL_PREFIX=/app -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
0.455 CMake Error: The current CMakeCache.txt directory /build/build_release/CMakeCache.txt is different than the directory /home/user/svetit/src/back/space/build_release where CMakeCache.txt was created. This may result in binaries being created in the wrong place. If you are not sure, reedit the CMakeCache.txt
0.455 CMake Error: The source "/build/CMakeLists.txt" does not match the source "/home/user/svetit/src/back/space/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
0.456 make: *** [Makefile:25: cmake-release] Error 1
```
Но добавление build_release/ в dockerignore работает. Поправлю во всех мс.

Reply:
Готово (через dockerignore).